src/widget/wtrackmenu.h

```diff
@@ -0,0 +1,259 @@
+    void updateSelectionCrates(QWidget* pWidget);
+
+    void showTrackInfo(QModelIndex index);
+    void showTrackInfo(TrackPointer pTrack);
+    void showDlgTagFetcher(QModelIndex index);
+    void showDlgTagFetcher(TrackPointer pTrack);
+
+    void addToAutoDJ(PlaylistDAO::AutoDJSendLoc loc);
+
+    void lockBpm(bool lock);
+
+    void loadSelectionToGroup(QString group, bool play = false);
+    void clearTrackSelection();
+
+    // Selected tracks
+    TrackPointerList m_pTrackPointerList;
+    QModelIndexList m_pTrackIndexList;
+
+    TrackModel* m_pTrackModel{};
+
+    const ControlProxy* m_pNumSamplers{};
+    const ControlProxy* m_pNumDecks{};
+    const ControlProxy* m_pNumPreviewDecks{};
+
+    // Submenus
+    QMenu* m_pLoadToMenu{};
+    QMenu* m_pDeckMenu{};
+    QMenu* m_pSamplerMenu{};
+    QMenu* m_pPlaylistMenu{};
+    QMenu* m_pCrateMenu{};
+    QMenu* m_pMetadataMenu{};
+    QMenu* m_pMetadataUpdateExternalCollectionsMenu{};
+    QMenu* m_pClearMetadataMenu{};
+    QMenu* m_pBPMMenu{};
+    QMenu* m_pColorMenu{};
+    WCoverArtMenu* m_pCoverMenu{};
+
+    // Reload Track Metadata Action:
+    QAction* m_pImportMetadataFromFileAct{};
+    QAction* m_pImportMetadataFromMusicBrainzAct{};
+
+    // Save Track Metadata Action:
+    QAction* m_pExportMetadataAct{};
+
+    // Load Track to PreviewDeck
+    QAction* m_pAddToPreviewDeck{};
+
+    // Send to Auto-DJ Action
+    QAction* m_pAutoDJBottomAct{};
+    QAction* m_pAutoDJTopAct{};
+    QAction* m_pAutoDJReplaceAct{};
+
+    // Remove from table
+    QAction* m_pRemoveAct{};
+    QAction* m_pRemovePlaylistAct{};
+    QAction* m_pRemoveCrateAct{};
+    QAction* m_pHideAct{};
+    QAction* m_pUnhideAct{};
+    QAction* m_pPurgeAct{};
+
+    // Show track-editor action
+    QAction* m_pPropertiesAct{};
+
+    // Open file in default file browser
+    QAction* m_pFileBrowserAct{};
+
+    // BPM feature
+    QAction* m_pBpmLockAction{};
+    QAction* m_pBpmUnlockAction{};
+    QAction* m_pBpmDoubleAction{};
+    QAction* m_pBpmHalveAction{};
+    QAction* m_pBpmTwoThirdsAction{};
+    QAction* m_pBpmThreeFourthsAction{};
+    QAction* m_pBpmFourThirdsAction{};
+    QAction* m_pBpmThreeHalvesAction{};
+
+    // Track color
+    WColorPickerAction* m_pColorPickerAction{};
+
+    // Clear track metadata actions
+    QAction* m_pClearBeatsAction{};
+    QAction* m_pClearPlayCountAction{};
+    QAction* m_pClearMainCueAction{};
+    QAction* m_pClearHotCuesAction{};
+    QAction* m_pClearIntroCueAction{};
+    QAction* m_pClearOutroCueAction{};
+    QAction* m_pClearLoopAction{};
+    QAction* m_pClearWaveformAction{};
+    QAction* m_pClearKeyAction{};
+    QAction* m_pClearReplayGainAction{};
+    QAction* m_pClearAllMetadataAction{};
+
+    const UserSettingsPointer m_pConfig;
+    TrackCollectionManager* const m_pTrackCollectionManager;
+
+    QScopedPointer<DlgTrackInfo> m_pTrackInfo;
+    QScopedPointer<DlgTagFetcher> m_pTagFetcher;
+
+    QModelIndex currentTrackInfoIndex;
```

Comment:
```suggestion
    QModelIndex m_currentTrackInfoIndex;
```
Use `m_` prefix for member variables so it is easy to tell them apart from local variables.

Reply:
Sure. I am following the convention. It's just a residue from old code. I will change this.